### ISSUE TITLE
SideNav component is not navigable using keyboard

### ISSUE THREAD

**Issue by jrios** [TWExchangeSolutions/es-components#65]
Because there is no explicit usage an ``a`` or ``button`` within the ``SideNav.Item`` component, the navigation cannot be navigated or triggered using a keyboard. This is an accessibility concern for keyboard-only users.

**Comment by jrios** [comment]
Some kind of tabbing needs to be added to the ``SideNav.Item`` components, or a decision should be made about making those links or buttons.

**Comment by Darrken** [comment]
I think the best bet is to make the Sidenav.Item a button.

**Comment by jrios** [comment]
I am inclined to agree. The only real argument against it is that it could "link" to another section of the page, which makes it more appropriate to use a link. A prop could be used to control which type of element to use, but that might be a bit much.

**Comment by Darrken** [comment]
That's why I didn't initially choose one over the other, and left it open.  I feel silly for totally forgetting to check the keyboard accessibility, I had intended to.

**Comment by jrios** [comment]
Same, I didn't check it until I pulled it into another project. Live and learn, I guess. 

I'm all for making it a button. Maybe we should discuss it with the UX team and get their thoughts before we make an implementation decision.

**Comment by Darrken** [comment]
Yeah, I'll demo it for them and get their input.
